rllab/misc/tensor_summary.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import numpy as np
+import tensorflow as tf
+from tensorboard import summary as summary_lib
+from tensorboard.plugins.custom_scalar import layout_pb2
+
+from rllab.misc.console import mkdir_p
+
+
+class Summary:
+    def __init__(self):
+        self._summary_scale = tf.Summary()
```

Comment:
what is the purpose of this variable? is it for scalars?

Reply:
i.e. should it be called `self._scalar_summary`? If so, you could shorten it to `self._scalars`